### ISSUE TITLE
Lantern Pro - Internet Blocked when I turn Lantern on

### ISSUE THREAD

**Issue by smojos** [getlantern/lantern#5505]
Hey, this is a new problem for me:

I work in a censored location in Afr#c#, so I got Lantern. Bought the pro version because I was very happy with the results. Now, just a week later, however, I'm experiencing a serious issue:

When I initially turn on Lantern it seems to work, I get uncensored internet acces. However, after a little while (sometimes a few minutes, sometimes a few hours) though the internet stops working. It connects but says "internet not available". When I turn Lantern off and restart the connection I get (censored) internet again, and turning Lantern back on results in internet blockage.

What's going on? Is Lantern being detected and blocked, or is this an issue with my computer?

**Comment by joesis** [comment]
@smojos To be clear, the Internet connection didn't recover even if you quitted Lantern? If so, it must some bug being triggered at the OS level. Lantern doesn't have the power to disable your network connection.

What's the exact OS version you are using? Did you upgrade your OS or install certain software recently?

**Comment by smojos** [comment]
Correct, internet did not recover when I quit Lantern. Does seem to recover after I wait... like right now, its a day later and I've restarted my computer several times and used my internet stick with a different computer.

My OS is Windows Home 10, V1511, Build 10586.679 (64bit)

I did not upgrade my OS or install software recently, other than Lantern... the first time I tried to intall Lantern I did get that problem (resolved on another thread) where Windows thought it was a trojan virus, but I downloaded the github lantern install .exe and that solved that.

**Comment by smojos** [comment]
Update: restarting my computer seems to reset the problem. I can then access the internet for a while.

**Comment by myleshorton** [comment]
OK thanks @smojos -- please let us know if the problem re-occurs! Also, if you go to http://whatismyipaddress.com/, what does it show you? Lantern uses UDP in some cases, and I suppose it's possible your router has some sort of issue with the UDP traffic.
